pages/common/atrm.md

```diff
@@ -0,0 +1,11 @@
+# atrm
+
+> Remove jobs scheduled by at or batch commands.
+
+- Remove job number 10:
+
+`atrm 10`
+
+- Remove many jobs separated by space:
```

Comment:
The wording here is a bit awkward. Perhaps a colon would help things here? I'm not sure.